server/models/user.js

```diff
@@ -26,10 +26,16 @@ module.exports = function (sequelize, DataTypes) {
       type: DataTypes.DATE,
       allowNull: true
     },
-    'isAdmin': {
-      type: DataTypes.BOOLEAN,
+    isAdmin: {
+      type: DataTypes.VIRTUAL,
+      get: function () {
+        return this.getDataValue('role') === 'admin'
+      }
+    },
+    role: {
+      type: DataTypes.STRING,
       allowNull: false,
-      defaultValue: false
+      defaultValue: 'user'
```

Comment:
Should add [validation](http://docs.sequelizejs.com/manual/tutorial/models-definition.html#validations) for the possible values